clippy_lints/src/utils/ast_utils.rs

```diff
@@ -10,6 +10,28 @@ use rustc_ast::{self as ast, *};
 use rustc_span::symbol::Ident;
 use std::mem;
 
+pub mod ident_iter;
+pub use ident_iter::IdentIter;
+
+pub fn is_useless_with_eq_exprs(kind: BinOpKind) -> bool {
```

Comment:
This could probably benefit from a local `use BinOpKind::*;`.

Reply:
Agreed.